src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/SolutionExplorer/NuGetWorkspaceCommandHandler.cs

```diff
@@ -15,15 +15,16 @@ namespace NuGet.VisualStudio.OnlineEnvironment.Client
     /// </summary>
     internal class NuGetWorkspaceCommandHandler : IWorkspaceCommandHandler
     {
-        private readonly JoinableTaskContext _taskContext;
-        private readonly IAsyncServiceProvider _serviceProvider;
-
-        private static bool IsNuGetFunctionalityAvailable = false;
```

Comment:
Yikes... so this was never used at all, ie, dead code? In your logic below, do we need to verify NuGet is in a certain state?

Reply:
I moved it down into command handlers. 
Didn't want to stub out a command handler with nothing inside when I merged the first PR.